target/ppc/translate/fp-impl.c.inc

```diff
@@ -1473,26 +1297,83 @@ static void gen_stfqx(DisasContext *ctx)
     tcg_temp_free_i64(t1);
 }
 
-static bool trans_PLFS(DisasContext *ctx, arg_PLFS *a)
+/*            Floating-point Load/Store Instructions                         */
+static bool do_lsfpsd(DisasContext *ctx, int rt, int ra, TCGv displ,
+                      bool update, bool store, bool single)
 {
     TCGv ea;
     TCGv_i64 t0;
-    arg_D d;
-    if (!resolve_PLS_D(ctx, &d, a)) {
+    REQUIRE_FPU(ctx);
```

Comment:
I think you need to check a insn flag first:
```suggestion
    REQUIRE_INSNS_FLAGS(ctx, FLOAT);
    REQUIRE_FPU(ctx);
```

Reply:
Done!